pallene/lexer.lua

```diff
@@ -70,15 +71,15 @@ do
 
     local contents = (-matching_close * P(1)) ^0
 
-    longstring = (
+    long_string = (
         open * (
             C(contents) * close +
             T("UnclosedLongString")
         )
-    ) / function(contents_str) return contents_str end -- hide the group captures
+    ) / function(contents_str) return contents_str end --hide the group captures
```

Comment:
I'd rather undo this one.

Reply:
If undone, 80 chars will be exceed in the line. Do I move the comment to the line before or after? Or leave it with 81 chars?